sky/packages/sky/lib/src/widgets/input.dart

```diff
@@ -22,40 +22,42 @@ class Input extends StatefulComponent {
 
   Input({
     GlobalKey key,
+    this.value : '',
     this.placeholder,
     this.onChanged,
     this.keyboardType : KeyboardType_TEXT
   }): super(key: key);
 
   int keyboardType;
+  String value;
   String placeholder;
   StringValueChanged onChanged;
 
-  String _value = '';
   EditableString _editableValue;
   KeyboardHandle _keyboardHandle = KeyboardHandle.unattached;
 
   void initState() {
     _editableValue = new EditableString(
-      text: _value,
+      text: value,
       onUpdated: _handleTextUpdated
     );
     super.initState();
   }
 
   void syncConstructorArguments(Input source) {
+    value = source.value;
```

Comment:
This change isn't correct.  We don't want to copy this constructor argument because it only has an effect in initState